active-response/restart.sh

```diff
@@ -45,9 +45,9 @@ fi
 if command -v systemctl > /dev/null 2>&1; then        
     systemctl restart wazuh-$TYPE
 elif command -v service > /dev/null 2>&1; then        
-    service restart wazuh-$TYPE
+    service wazuh-$TYPE restart
 elif command -v update-rc.d > /dev/null 2>&1; then        
-    update-rc.d restart wazuh-$TYPE
+    update-rc.d wazuh-$TYPE restart
```

Comment:
This line isn't necessary. `update-rc.d` is used to install/remove System-V style init script links. In addition, `update-rc.d` doesn't have any `restart` option so if the script executes this line, it will fail and exit without restarting the service.

Reply:
Done!